designdocs/privacy-design.md

```diff
@@ -0,0 +1,163 @@
+*   If an input to [getPose](https://immersive-web.github.io/webxr/#dom-xrframe-getpose) includes data generated by sensors, at least one of the following conditions must be met; the user agent may choose the approach that provides the best user experience:
+    *   User consent must have been obtained, _or_
+    *   Any transform data generated by sensors must be [quantized](https://github.com/immersive-web/privacy-and-security/blob/master/POSE-AND-ENVIRONMENT.md#rounding-quantization-or-fuzzing) to prevent sensor fingerprinting, _or_
+    *   The user agent must otherwise ensure that the underlying device sensors are not susceptible to sensor fingerprinting.
+
+#### Creating XRReferenceSpace
+The user agent must verify that all mandatory conditions are satisfied to ensure it can create an [XRReferenceSpace](https://immersive-web.github.io/webxr/#xrreferencespace-interface) and expose [XRReferenceSpace](https://immersive-web.github.io/webxr/#xrreferencespace-interface) data within a given [active document](https://html.spec.whatwg.org/multipage/browsers.html#active-document).
+
+The **_mandatory conditions_** for creating an XRReferenceSpace are the following:
+*   User consent must have been obtained before creating a [bounded](https://immersive-web.github.io/webxr/#dom-xrreferencespacetype-bounded) or [unbounded](https://immersive-web.github.io/webxr/#dom-xrreferencespacetype-unbounded) reference space. 
+*   User consent is required before creating a [floor-level](https://immersive-web.github.io/webxr/#dom-xrreferencespacetype-floor-level) reference space type, if the floor level will reflect the real-world location of the user’s floor.
+*   When creating a [floor-level](https://immersive-web.github.io/webxr/#dom-xrreferencespacetype-floor-level) or [bounded](https://immersive-web.github.io/webxr/#dom-xrreferencespacetype-bounded) reference space type, if the origin of the reference space reflects the real-world location of the user’s floor, or if the floor level is emulated and set by the user to a non-default value, then the coordinates of the origin must be [rounded sufficiently](https://github.com/immersive-web/privacy-and-security/blob/master/POSE-AND-ENVIRONMENT.md#rounding-quantization-or-fuzzing) to prevent fingerprinting using floor level. Rounding to the nearest 1cm is suggested.
+*   When creating a [bounded](https://immersive-web.github.io/webxr/#dom-xrreferencespacetype-bounded) reference space the [boundsGeometry](https://immersive-web.github.io/webxr/#dom-xrboundedreferencespace-boundsgeometry) must be rounded sufficiently to prevent fingerprinting the user’s bounds. The region represented by the rounded bounds geometry must be a subset of the original bounds. Rounding to the nearest 5cm is suggested.
```

Comment:
>Rounding to the nearest 5cm is suggested.

Suggested?  Or the minimum required?  Either way, I'm not certain this is really enough to mitigate the concern on systems like Windows MR which allow the user to walk the boundary to define it.

Reply:
> a user agent may wish to acquire user consent in cases where the bounds are likely to be truly unique. 

This feels a bit at odds with the statement above about IPD fingerprinting, tbh.  I mean when you're drawing a weird shaped bounds (which the Quest also now supports) you're going to end up fingerprinted.

A possible mitigation would be a smaller random polygon inside the bounds?  But.. ick.